sync-eni-max-pods.yaml

```diff
@@ -0,0 +1,40 @@
+name: '[Sync] Update eni-max-pods.txt'
+on:
+  workflow_dispatch:
+  schedule:
+    # once a day
+    - cron: '0 0 * * *'
+permissions:
+  id-token: write
+  contents: write
+  pull-requests: write
+jobs:
+  update-max-pods:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: ${{ secrets.AWS_REGION }}
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+      - uses: actions/checkout@v3
+        with:
+          repository: awslabs/amazon-eks-ami
+          ref: refs/heads/master
+          path: amazon-eks-ami/
+      - uses: actions/checkout@v3
+        with:
+          repository: aws/amazon-vpc-cni-k8s
+          ref: refs/heads/master
+          path: amazon-vpc-cni-k8s/
+      - run: |
+          #!/usr/bin/env bash
+          set -o errexit
+          cd amazon-vpc-cni-k8s/
+          make generate-limits
```

Comment:
Could you please link the change for avoiding update the timestampe as well?

Reply:
https://github.com/aws/amazon-vpc-cni-k8s/commit/4e5b5b35ae72033087eae9e14e597f33d020ba8a

Reply:
The first time this runs, it will open a PR because the timestamp has been removed; but after that, it will only open a PR if the max pods values actually change.